README.md

```diff
@@ -121,7 +123,7 @@ npm start
 
 ## 🔭 Future Features <a name="future-features"></a>
 
```

Comment:
- [ ] Kindly consider removing the `Live Demo` section altogether until you deploy the website